index.html

```diff
@@ -22,22 +27,70 @@
             let protocol = new pmtiles.Protocol();
             maplibregl.addProtocol("pmtiles",protocol.tile);
 
-            let PMTILES_URL = "https://mizmay.github.io/RentAffordability/RentAffordability-OneBr.pmtiles";
+            let PMTILES_URL = "./RentAffordability-OneBr.pmtiles";
 
             const p = new pmtiles.PMTiles(PMTILES_URL);
 
             // this is so we share one instance across the JS code and the map renderer
             protocol.add(p);
 
             // we first fetch the header so we can get the center lon, lat of the map.
-            p.getHeader().then(h => {
+            p.getHeader().then(async h => {
                 const map = new maplibregl.Map({
                     container: 'map',
                     zoom: h.minZoom-.5, // hack to make this fit well in an iframe
                     center: [h.centerLon, h.centerLat],
-                    style: "https://mizmay.github.io/RentAffordability/mizmay_alidade_smooth.json"
+                    style: "./mizmay_alidade_smooth.json"
```

Comment:
Absolutely yes, this should allow for running a local server and developing iterating on the stylesheet locally, great.